fcli-core/fcli-common/src/main/java/com/fortify/cli/common/rest/unirest/config/UrlConfig.java

```diff
@@ -35,15 +35,7 @@ public static final UrlConfig from(IUrlConfig other) {
     }
     
     public static final UrlConfigBuilder builderFrom(IUrlConfig other) {
-        UrlConfigBuilder builder = UrlConfig.builder();
-        if ( other!=null ) {
-            builder = builder
-                .url(other.getUrl())
-                .insecureModeEnabled(other.isInsecureModeEnabled())
-                .connectTimeoutInMillis(other.getConnectTimeoutInMillis())
-                .socketTimeoutInMillis(other.getSocketTimeoutInMillis());
-        }
-        return builder;
+        return builderFrom(other);
```

Comment:
Previous implementation also did `.url(other.getUrl())`, new implementation must do the same (including null-check)